lib/init/add-contributors-list.test.js

```diff
@@ -1,27 +1,25 @@
 import test from 'ava';
-import {addContributorsList} from './init-content';
+import {addContributorsList, setContentLanguage} from './init-content';
 
 test('should insert list under contributors section', t => {
   const content = [
     '# project',
     '',
     'Description',
     '',
-    '## Contributors',
+    '## Contributors<div id=\'contributors\'></div>',
```

Comment:
That is the idea.
Mainly it was not to add another `language handling` on the link of the badge.

I thought it's no harm... Should I change it?